modules/serverless-kafka-broker.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-kafka-developer.adoc
+// * serverless/develop/serverless-using-brokers.adoc
+
+[id="serverless-kafka-broker_{context}"]
+= Creating a Kafka broker
+
+.Prerequisites
+
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource are installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Kafka-based broker as a YAML file:
++
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    eventing.knative.dev/broker.class: Kafka <1>
+  name: example-kafka-broker
+spec:
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config <2>
+    namespace: knative-eventing
+----
+<1> The broker class. If not specified, brokers use the default class as configured by cluster administrators.
```

Comment:
Should the items 1&2 in the YAML above be formatted as user-replaceable values?

Reply:
Changed this text a bit to make it clearer that a user should use these values.